### ISSUE TITLE
API 'variantOutput.getPackageLibrary()' is obsolete 

### ISSUE THREAD

**Issue by ThisIsLeon** [kezong/fat-aar-android#105]
INFO: API 'variantOutput.getPackageLibrary()' is obsolete and has been replaced with 'variant.getPackageLibraryProvider()'.
It will be removed at the end of 2019.

**Comment by acortelyou** [comment]
Stacktrace implicates line 42 of `FatLibraryPlugin.groovy`:

```
Caused by: groovy.lang.MissingPropertyException: No such property: libraryVariants for class: java.lang.String
        at com.kezong.fataar.FatLibraryPlugin$_apply_closure1.doCall(FatLibraryPlugin.groovy:42)
```

 https://github.com/kezong/fat-aar-android/blob/16746744df9db0ac5e632f679195acc49fbc7b47/source/src/main/groovy/com/kezong/fataar/FatLibraryPlugin.groovy#L42-L44

**Comment by kezong** [comment]
Fixed it in 1.2.12
